src/exampleTypes/TagsQuery/index.js

```diff
@@ -53,16 +54,20 @@ let TagsQuery = ({
     >
       <GridItem height={2} place="center stretch">
         <TagsInput
-          splitCommas
+          splitCommas={splitCommas}
           sanitizeTags={sanitizeTags}
           wordsMatchPattern={wordsMatchPattern}
           tags={_.map(tagValueField, node.tags)}
+          // addTag: Called on every tag being added via the input. Use to decorate the tag into object form
           addTag={tag => {
-            tree.mutate(node.path, {
-              tags: [...node.tags, { [tagValueField]: tag, distance: 3 }],
-            })
-            onAddTag(tag)
+            let tagObject = { [tagValueField]: tag, distance: 3 }
+            onAddTag(tagObject)
+            return tagObject
```

Comment:
I think we can simplify this and just have a single addTags method, since both methods end up calling onTagsAdd anyway

Reply:
Refactored to use one method `addTags`.